server/src/main/java/com/cloud/api/ApiResponseHelper.java

```diff
@@ -5159,4 +5158,15 @@ public BucketResponse createBucketResponse(Bucket bucket) {
         populateAccount(bucketResponse, bucket.getAccountId());
         return bucketResponse;
     }
+
+    private String getDomainPath(String path){
+        StringBuilder domainPath = new StringBuilder();
+        if (StringUtils.isEmpty(path) || path.equals("/")){
+            domainPath.append("/");
+        }
+        else{
+            (domainPath.append(path)).deleteCharAt(domainPath.length() - 1);
```

Comment:
```suggestion
        if (StringUtils.isEmpty(path) || path.equals("/")) {
            domainPath.append("/");
        } else {
            (domainPath.append(path)).deleteCharAt(domainPath.length() - 1);
```
@isupersky please note the code style changes.